cdbot/cogs/admin.py

```diff
@@ -0,0 +1,55 @@
+import re
+from discord import Member
+from discord.ext.commands import Bot, Cog, Context, command
+
+from cbot.constants import (
+    PLACEHOLDER_NICKNAME, NICKNAME_PATTERNS
+)
+
+def checkName(nick):
+    result = False
+    for i in NICKNAME_PATTERNS:
```

Comment:
This is a good way to do this but I'm thinking whether it would be more Pythonic to use a for/else construction? In this case I see no obvious advantage either way (except maybe some kind of minor efficiency improvement) but I'll leave it to the more experienced for further commentary.

Reply:
It could be replaced by a list comprehension:
``
return any(re.match(i,nick) for i in NICKNAME_PATTERNS)
``